docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:9.9-slim
+FROM marketplace.gcr.io/google/debian9
```

Comment:
We would like our builds to be reproducible, so we can't just pick up the latest image. And the images in that repo don't seem to have meaningful version tags.

Let's update to `debian:9.13-slim` instead.

Reply:
Done